README.md

```diff
@@ -9,124 +9,136 @@ The following repository contains:
+6. Docker daemon for example Docker Desktop:
+   - [MacOS](https://docs.docker.com/desktop/install/mac-install/)
+   - [Windows](https://docs.docker.com/desktop/install/windows-install/)
+   - [Linux](https://docs.docker.com/desktop/install/linux-install/)
+7. [NumPy](https://numpy.org/install/)
 
-- numpy
+> **TIP:** If you're a macOS user, you can also use `brew` to install NumPy:
 
-  ```bash
-  pip3 install numpy
-  ```
+```bash
+brew install numpy
+```
 
-  For **brew** users:
+8. [OpenCV-Python](https://pypi.org/project/opencv-python/)
 
-  ```bash
-  brew install numpy
-  ```
+9. [Imutils](https://pypi.org/project/imutils/)
 
-- imutils
+Check if you have installed all of the [prerequisites](https://github.com/HackYourCareer/SmartKickers#prerequisites).
 
-  ```bash
-  pip3 install imutils
-  ```
+> **TIP:** If you're a macOS user, you can run the [`requirementsCheck.sh`](requirementsCheck.sh) script to check for any missing dependencies. Note that the script looks for OpenCV, NumPy, Imutils, and the Ximea driver **only after you have successfully installed Python!** If you don't have Python, it doesn't print missing Python-dependent packages except Python itself.
+
+```bash
+sudo ./requirementsCheck.sh
+```
 
 ## Installation
 
-We are using macOS to build and run everything locally.
+1. Create a new directory and navigate to it in a terminal.
 
-1. Clone the repositories.
+```bash
+mkdir smartkickers
+cd smartkickers
+```
 
-   ```bash
-   git clone https://github.com/HackYourCareer/SmartKickers.git
-   git clone https://github.com/HackYourCareer/SmartKickersAI.git
-   ```
+2. Clone the repository.
 
-2. Run the [`requirementsCheck.sh`](requirementsCheck.sh) script to check if your system fulfils all requirements.
+```bash
+git clone https://github.com/HackYourCareer/SmartKickersAI.git
+```
 
-   The script will look for open-cv, numpy, imutils, ximer driver only after successfully installing python! So if you don't have python it won't print missing python dependent packages except python itself.
+3. Pull frontend image from GCR
 
-   ```bash
-   cd SmartKickers
-   ./requirementsCheck.sh
-   ```
+```bash
+docker pull ghcr.io/hackyourcareer/smartkickers-frontend:latest
+```
 
-   > **NOTE:** Be sure to give permission for the script to execute.<br/>
+4. Pull backend image from GCR
 
-   ```bash
-   sudo chmod +x requirementsCheck.sh
-   ```
+```bash
+docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
+```
 
-3. Install missing dependencies.
+## Launching
 
-4. In folder [`SmartKickers`](SmartKickers) run docker-compose build.
+> **NOTE:** Configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
 
-   ```bash
-   docker-compose build
-   ```
+### Launching the server and web application
 
-## Launching
+- Run pulled containers
+
+  - In a terminal run:
```

Comment:
- An article is missing.
- If you have only one step, you don't need to format it as a list. Just type it as a regular sentence. 
  - However, the sub-elements are sequential steps (should be performed in a given order), so they should be an ordered (numbered) list. I don't think it's necessary to put them as sub-elements, as pulling the containers is all you have to do to launch. 
-  Because you have further subsections, consider adding info what you're launching here. (Like: `Run the pulled containers to launch the application interface.`)

```suggestion
Run the pulled containers.

1. In a terminal, run:
```